automatix/command.py

```diff
@@ -105,9 +113,9 @@ def _execute(self, interactive: bool = False, force: bool = False):
         if self.get_type() == 'manual' or interactive:
             self.env.LOG.debug('Ask for user interaction.')
 
-            answer = self._ask_user(question='[MS] Proceed?', allowed_options=['p', 's', 'R', 'a'])
+            answer = self._ask_user(question='[MS] Proceed?', allowed_options=['p', 'T', 's', 'R', 'a'])
             # answers 'a', 'c' and 'R' are handled by _ask_user, 'p' means just pass
-            if answer == 's':
+            if answer == 's' or self.get_type() == 'manual':
```

Comment:
Hmm github does not to update Pullrequests, if the main branch changes? This should be already on `main`.

Reply:
OK, found it. Had to update my branches via "sync fork".